doc/source/user_guide/migration.rst

```diff
@@ -0,0 +1,215 @@
+            return self.name
+
+
+Then you need to create a ``MeshObjectPlot`` instance that relates the PyVista mesh with your custom object.
+
+.. code-block:: python
+
+    from ansys.tools.visualization_interface import MeshObjectPlot
+
+    custom_object = CustomObject()
+    mesh_object_plot = MeshObjectPlot(
+        custom_object=custom_object,
+        mesh=custom_object.get_mesh(),
+    )
+
+With this, you can use the Ansys Tools Visualization Interface plotter to visualize your custom object. It enables interactivity such as picking and hovering.
+
+
+Customizing the PyVista backend
+-------------------------------
+
+You can customize the backend of the Ansys Tools Visualization Interface plotter to enable or turn off certain functionalities. For example,
+if you want to enable picking, you can do it as follows:
+
+.. code-block:: python
+
+    from ansys.tools.visualization_interface import Plotter
+    from ansys.tools.visualization_interface.backends import PyVistaBackend
+
+    backend = PyVistaBackend(allow_picking=True)
+
+    # Create a plotter
+    pl = Plotter(backend=backend)
+
+    # Add the MeshObjectPlot instance to the plotter
+    pl.plot(mesh_object_plot)
+
+    # Show the plotter
+    pl.show()
+
+If you want to go further and customize the backend even more, you can create your own backend by inheriting from the ``PyVistaBackendInterface`` class
+and implementing the required methods. You can find more information about this in the backend documentation:
+
+.. code-block:: python
+
+    @abstractmethod
+    def plot_iter(self, plottable_object: Any, name_filter: str = None, **plotting_options):
+        """Plot one or more compatible objects to the plotter.
+
+        Parameters
+        ----------
+        plottable_object : Any
+            One or more objects to add.
+        name_filter : str, default: None.
+            Regular expression with the desired name or names  to include in the plotter.
+        **plotting_options : dict, default: None
+            Keyword arguments. For allowable keyword arguments, see the
+            :meth:`Plotter.add_mesh <pyvista.Plotter.add_mesh>` method.
+
+        """
+        pass
+
+
+    @abstractmethod
+    def plot(self, plottable_object: Any, name_filter: str = None, **plotting_options):
+        """Plot a single object to the plotter.
+
+        Parameters
+        ----------
+        plottable_object : Any
+            Object to add.
+        name_filter : str
+            Regular expression with the desired name or names to include in the plotter.
+        **plotting_options : dict, default: None
+            Keyword arguments. For allowable keyword arguments, see the
+            :meth:`Plotter.add_mesh <pyvista.Plotter.add_mesh>` method.
+
+        """
+        pass
+
+
+The rest of the methods are implemented for you. This ensures that while you can customize what you need for plotting, the rest of the functionalities still work as expected.
```

Comment:
```suggestion
The rest of the methods are implemented for you. This ensures that while you can customize what you need for plotting, the rest of the functionalities still work as expected. For more information, see the backend documentation.
```
Not sure that the last sentence here belongs here--but it was awkward having it directly before the code.